jobs/config.json

```diff
@@ -2890,6 +2909,25 @@
       "sig-instrumentation"
     ]
   },
+  "ci-kubernetes-e2e-gce-kubedns-performance": {
+    "args": [
+      "--cluster=gce-kubedns-performance",
+      "--env=CLUSTER_DNS_CORE_DNS=false",
+      "--extract=ci/latest",
+      "--gcp-master-image=gci",
+      "--gcp-node-image=gci",
+      "--gcp-nodes=3",
+      "--gcp-zone=us-east1-b",
+      "--provider=gce",
+      "--test_args=--ginkgo.focus=\\[Feature:PerformanceDNS\\] --gather-resource-usage=master --gather-metrics-at-teardown=master",
+      "--timeout=10m",
```

Comment:
10m is too low I think (this time includes creating cluster, running test, copying logs, bringing down, etc). Please set it sth higher, like 60m maybe.